.ci/install_crio.sh

```diff
@@ -18,9 +18,13 @@ set -e
 
 cidir=$(dirname "$0")
 
+source "${cidir}/../test-versions.txt"
+
 echo "Get CRI-O sources"
 go get -d github.com/kubernetes-incubator/cri-o || true
 pushd $GOPATH/src/github.com/kubernetes-incubator/cri-o
+git fetch
```

Comment:
Ah, OK. I understand. I have similar niggles trying to make the static checker scripts runnable by anybody from anywhere that I need to get fixed.....
It is a tricky one. Sure, it would be nice to be able to run the scripts under any circumstance, but they do live in the `.ci` dir, so I don't think it is out of order for them to fail in strange setups.
Ultimately, I don't think we have to cater for hand runs if it costs us. In this case, I don't think the `fetch` will cost us much - so, if @sboeuf agrees, I would ask we have a clear comment added above the fetch explaining why it is there - OK?